src/github.com/matrix-org/rageshake/submit.go

```diff
@@ -93,12 +95,20 @@ func (s *submitServer) saveReport(ctx context.Context, p payload) error {
 	//  "bugreport-20170115-112233-N.log.gz" => oldest log
 	t := time.Now().UTC()
 	prefix := t.Format("2006-01-02/150405")
-	summary := fmt.Sprintf(
-		"%s\n\nNumber of logs: %d\nVersion: %s\nUser-Agent: %s\n", p.Text, len(p.Logs), p.Version, p.UserAgent,
+
+	var summaryBuf bytes.Buffer
+	fmt.Fprintf(
+		&summaryBuf,
+		"%s\n\nNumber of logs: %d\nApplication: %s\nVersion: %s\nUser-Agent: %s\n",
```

Comment:
IMO it's cognitively easier to grok what:
```go
summaryBuf.WriteString(fmt.Sprintf("Hello %s", name))
```
is doing than:
```go
fmt.Fprintf(&summaryBuf, "Hello %s", name)
```

As for `buf.String()` - if you want a byte output then fair enough, for some reason I thought you wanted a string.